### PR TITLE
PD-3848 update packages remove npm refs

### PR DESCRIPTION
**Pull request by iartemiev**
## What's this PR do?
* Updated Webpack and webpack-dev-server to their latest versions.
* Made adjustments to `webpack.config.js`
* Replaced call to `npm` with `yarn` in `rundevserver` in `dockerdev.sh`
* deleted npm-shrinkwrap

### CODE REVIEW

**Comment by mwarnsleyDE** [comment]
Everything looks good to me @iartemiev except I can't judge some of the python files or anything like that so those would be on the backend realm lol.

File: gulp/src/reporting/components/ReportingIntention.jsx [hunk 1]
```diff
@@ -23,32 +23,17 @@ class ReportingIntention extends Component {
 
   renderIntention(choice, isGroup) {
     return (
-      <li
-          key={choice.value}
-          className={
-            isGroup ? 'report-item clickable' : 'report-site clickable'}>
-        <div
-          onClick={() => this.handleLinkClick(choice)}
-          className="reports-link">
```
Comment (koonsies) [inline gulp/src/reporting/components/ReportingIntention.jsx]: What are these changes? They don't seem related to the PR description.

Comment (koonsies) [reply gulp/src/reporting/components/ReportingIntention.jsx]: Ahh.. Nevermind. Linting changes?

Comment (iartemiev) [reply gulp/src/reporting/components/ReportingIntention.jsx]: It was just some stuff that got auto-formatted... I added them accidentally.

**Comment by aljordan82** [comment]
I can get this to work, I am getting errors when running `dk rundevserver`

**Comment by koonsies** [comment]
Mind pasting the error? It seems to work fine for me so far.

**Comment by mwarnsleyDE** [comment]
@aljordan82 it worked for me. 95% of the time it's user error. LOL

**Comment by koonsies** [comment]
dky ran fine for me even after deleting my node_modules directory. Verifying that runserver works as well.

**Comment by aljordan82** [comment]
Clearing yarn cache solved my issue, however, the devserver is randomly restarting and there are constant XHR requests and high cpu usage on my machine.
screen shot 2017-10-03 at 11 58 53 am

**Comment by aljordan82** [comment]
Added DNM because I am experiencing High CPU usage and constant reloading.

**Comment by iartemiev** [comment]
@aljordan82 - sounds good, we’ll try to get to the bottom of it tomorrow!

**Comment by tdphillips** [comment]
Is this still being worked on?

**Comment by tdphillips** [comment]
If future work happens, feel free to reopen.
